rules/test/snapshot/BundleBridge.mm

```diff
@@ -0,0 +1,14 @@
+#import "BundleBridge.h"
+
+@implementation NSBundle (BazelBundleBridge)
+
+- (NSString *)bazelRunfilePathForResource:(NSString *)resource
+{
+    // TODO Determine:
+    // Does the existing C++ code solving our problem, if so that is awesome and
+    // potentially  more robust than a simple implementation that we do for
```

Comment:
On this line of code, we could confirm if this code is setup to work E2E 👍 
https://github.com/bazelbuild/bazel/blob/master/tools/cpp/runfiles/runfiles_src.h#L28